java/gazelle/testdata/attribute_setting/expectedStderr.txt

```diff
@@ -1 +1,2 @@
 [90m12:00AM[0m [31mWRN[0m [1mjava/gazelle/resolve.go:XXX[0m[36m >[0m Unable to find package for import in any dependency [36mfrom rule=[0m//src/test/com/example/myproject [36mpackage=[0mcom.example.myproject
+[90m12:00AM[0m [31mWRN[0m [1mjava/gazelle/resolve.go:XXX[0m[36m >[0m Unable to find package for import in any dependency [36mfrom rule=[0m//src/test/com/onlyflaky/myproject:RandomTest [36mpackage=[0mcom.example.onlyflaky.myproject
```

Comment:
Are we really expecting a new line of output?

Reply:
We actually are, but the larger PR this is all in preparation for will remove it :)